.github/workflows/CI.yaml

```diff
@@ -172,3 +172,16 @@ jobs:
       working-directory: ./${{ matrix.config.working-dir }}
       run: |
         make build-and-push-image TAG=dev-${{ env.DATETIME }}
+
+    - name: Generate release.yaml
+      id: generate_release_yaml
+      run: |
+        cd operator
+        make controller-gen release-manifests TAG=dev-${{ env.DATETIME }}
```

Comment:
we miss function runtime and scheduler

Reply:
The generation of the scheduler manifest was added in the last commit. 
The functions-runtime doesn't have any manifests to publish